src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryConfiguration.java

```diff
@@ -70,6 +70,7 @@ public class LibraryConfiguration extends AbstractDescribableImpl<LibraryConfigu
      * Library name.
      * Should match {@link Library#value}, up to the first occurrence of {@code @}, if any.
      */
+    @CheckForNull
```

Comment:
I think it would make more sense to treat this as `@NonNull`, throw an IAE going forward if someone attempts to save a nameless lib, and omit malformed entries when loading `GlobalLibraries` or `FolderLibraries`.